docs/http-api/tasks.rst

```diff
@@ -0,0 +1,114 @@
+Tasks
+=====
+
+Get list of tasks
+-----------------
+
+Request
+^^^^^^^
+
+.. code-block::
+
+  GET /tasks?project_id=1
```

Comment:
I thought that we will have something like `/project/<id>/tasks`
like described [here](https://www.moesif.com/blog/technical/api-design/REST-API-Design-Best-Practices-for-Sub-and-Nested-Resources/)
if you have a link (or your own argument) that proves me wrong, i am all ears

Reply:
I like this thing. In fact earlier version of this document (made in January) was based on this idea. Unfortunately this had several downsides in our case:
1. URLs are too long: `/projects/.../tasks/.../models/.../images`;
1. Unclear what to do with instances which "reference" both images and environments;
1. More suited for work with natural ids (in our case names: `/projects/<name>/tasks/<name>`, etc) and not synthetic ones: to retrieve a model with some id we do not need ids of project and task it belongs to but in this case we should either pass them anyway or have too much URL variations.